hoodie-client/src/main/java/com/uber/hoodie/index/hbase/HBaseIndex.java

```diff
@@ -162,20 +183,20 @@ private boolean checkIfValidCommit(HoodieTableMetaClient metaClient, String comm
               // iterator till we reach batch size
               if (statements.size() >= multiGetBatchSize || !hoodieRecordIterator.hasNext()) {
                 // get results for batch from Hbase
-                Result[] results = hTable.get(statements);
+                Result[] results = doGet(hTable, statements);
                 // clear statements to be GC'd
                 statements.clear();
                 for (Result result : results) {
                   // first, attempt to grab location from HBase
                   HoodieRecord currentRecord = currentBatchOfRecords.remove(0);
                   if (result.getRow() != null) {
                     String keyFromResult = Bytes.toString(result.getRow());
-                    String commitTs = Bytes
-                        .toString(result.getValue(SYSTEM_COLUMN_FAMILY, COMMIT_TS_COLUMN));
-                    String fileId = Bytes
-                        .toString(result.getValue(SYSTEM_COLUMN_FAMILY, FILE_NAME_COLUMN));
-                    String partitionPath = Bytes
-                        .toString(result.getValue(SYSTEM_COLUMN_FAMILY, PARTITION_PATH_COLUMN));
+                    String commitTs =
```

Comment:
Same here, is this intentional ? We want to avoid any changes which are not intentional.

Reply:
Imported the code style recommended, but some like these are not covered by it I guess since we there is no strict 1:1 mapping btw that and maven check-style. Sure, will undo this.